framework/src/play-jdbc/src/main/resources/reference.conf

```
@@ -96,7 +96,7 @@ play {
         # 3. A value less than zero will bypass any initial connection attempt, and the pool will start immediately
         #    while trying to obtain connections in the background. Consequently, later efforts to obtain a connection
         #    may fail.
-        initializationFailTimeout = 1
+        initializationFailTimeout = -1
```

So, the new default is NOT to fail fast.